src/server/verification/faceRecognition/faceRecognitionHelper.js

```diff
@@ -108,8 +115,9 @@ const Helper = {
   async enroll(zoomData: FormData): EnrollResult {
     try {
       let res = await ZoomClient.enrollment(zoomData)
-      log.debug('enroll result:', { res })
-      if (res.meta.ok) return res.data
+      let results = res.data
+      results && log.debug('enroll result:', { results })
```

Comment:
same no need for results && and add function name

Reply:
removed, commit ed4ee0da4d09f3e67a4e8523866aaeddd36ed7b2, will be resolved once pushed